server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1980,9 +1995,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 						Lumberjack.error("Error writing checkpoint", lumberjackProperties, error);
 					});
 				const checkpointReason = CheckpointReason[checkpointParams.reason];
-				lumberjackProperties.checkpointReason = checkpointReason;
-				const checkpointResult = `Writing checkpoint. Reason: ${checkpointReason}`;
-				Lumberjack.info(checkpointResult, lumberjackProperties);
+				this.apiCounter.incrementCounter(checkpointReason);
```

Comment:
Add an if (this.deliCheckpointMetricInterval > 0) { } around these lines too.

Reply:
Added.